pkg/apis/contrail/v1alpha1/swift_types.go

```diff
@@ -51,6 +51,13 @@ type SwiftList struct {
 	Items           []Swift `json:"items"`
 }
 
+// SetDefaultValues sets default values for swift resource parameters
+func (s *Swift) SetDefaultValues() {
+	if s.Spec.ServiceConfiguration.SwiftProxyConfiguration.SwiftServiceName == "" {
+		s.Spec.ServiceConfiguration.SwiftProxyConfiguration.SwiftServiceName = SwiftServiceName
+	}
+}
```

Comment:
So if you use helper function like this then use the same logic for keystone_types.go :)